src/routes/index.js

```diff
@@ -89,6 +89,52 @@ const getDevice = (device) => {
   });
 }
 
+async function queryGitHubReleasesApi(requestedInstallerPackage) {
+  return new Promise ((resolve, reject) => {
+    request.get({
```

Comment:
Please make a cache of this (10 min or so), github has an rate limit that we must follow. and if many users request this it will block the servers IP